onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -357,9 +357,10 @@ Status SequentialExecutor::Execute(const SessionState& session_state, const std:
                 << "\n";
 #endif
 
+      kernel_end_time = session_state.Profiler().Now();
```

Comment:
Same here. We best move kernel_end_time = session_state.Profiler().Now(); at the begin() of post processing block. 
i.e., before the CalculateTotalOutputSizes @ line 345.

Reply:
Done.